### PR TITLE
Code cleanup of HuntIR module

### PR DESCRIPTION
**Pull request by jokoho48**
Code cleanup of HuntIR module.

### CODE REVIEW

**Comment by Glowbal** [comment]
Seems like a very trivial clean up :+1:

File: addons/huntir/functions/fnc_cam.sqf [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
  * Return Value:
- * Nothing
+ * None
  *
  * Public: No
  */
 #include "script_component.hpp"
 
-PARAMS_1(_huntIR);
+parama ["_huntIR"];
```
Comment (SilentSpike) [inline addons/huntir/functions/fnc_cam.sqf]: typo

File: addons/huntir/functions/fnc_handleFired.sqf [hunk 2]
```diff
@@ -33,15 +33,16 @@ if (_ammo != "F_HuntIR") exitWith {};
...
         _huntir = createVehicle ["ACE_HuntIR", _position, [], 0, "FLY"];
         _huntir setPosATL _position;
         _huntir setVariable [QGVAR(startTime), ACE_time, true];
         [{
-            EXPLODE_1_PVT(_this select 0,_huntir);
+            params ["_args", "_idPFH"];
+            _args params ["_huntir"];
             if (isNull _huntir) exitWith {
-                [_this select 1] call CBA_fnc_removePerFrameHandler;
+                [_idPFH] call CBA_fnc_removePerFrameHandler;
```
Comment (SilentSpike) [inline addons/huntir/functions/fnc_handleFired.sqf]: This variable doesn't exist

Comment (jokoho48) [inline addons/huntir/functions/fnc_handleFired.sqf]: of cause L42?

Comment (SilentSpike) [inline addons/huntir/functions/fnc_handleFired.sqf]: Ignore that :laughing: I didn't see the inner `params`

**Pull request merged**